README.md

```diff
@@ -85,6 +85,48 @@ Get a list of repositories from a text file, structured as line-separated `owner
   </tr>
 </table>
 
+### Get repositories through Bizops
+Before executing any of the two commands below, either through `npx nori` interactive mode, or directly through the command line, make sure you set the env variable BIZ_OPS_API_KEY. To get the key, run /bifrost in any slack channel, which will create the key in https://developer.ft.com/portal/member.
+
+##### `nori team-repos`
+This operation gets all the repositories from the team you have selected.
+It shows a list of teams in customer products (obtained from Bizops) which you can select. 
+
+<table>
+  <tr>
+    <th align="right">Arguments</th>
+    <td>None</td>
+    <td><em>Teams to be selected when command is run</em></td>
+  </tr>
+  <tr>
+    <th align="right">Inputs</th>
+    <td colspan="2"><code>none</code></td>
+  </tr>
+  <tr>
+    <th align="right">Output</th>
+    <td colspan="2"><code>repos</code></td>
+  </tr>
+</table>
+
+##### `nori graphql-repos --file`
```

Comment:
```suggestion
##### `nori graphql-repos --file <file path>`
```
suggestion: I think it might make sense to add that you need to include the file path at this point.